### DISCUSSION TITLE
CI Resource Pack requested

### DISCUSSION THREAD

**Discussion by mickaelistria** [eclipse-m2e/m2e-core#1458] [General]
Hi,
As requested by @laeubi , Red Hat is assigning 1 CI Resource Pack for the Jenkins infra at Eclipse.org, https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3312 tracks the setup.

**Comment by laeubi** [comment]
@mickaelistria many thanks!

**Comment by HannesWell** [comment]
Thank you! And thanks Christoph for asking. 🎉
